app/views/course/achievement/achievements/_achievement.html.slim

```diff
@@ -24,9 +24,13 @@
 
   td
     div.btn-group
-      - if can?(:manage, achievement)
+      - if can?(:award, achievement)
```

Comment:
For `achievements#index`, the statement `@achievements = @achievements.includes(:conditions)` already preloads. Does this work?